notifier/pull.py

```diff
@@ -155,3 +170,29 @@ def generate_digest_content(users_by_id, from_dt, to_dt):
         resp = _http_post(api_url, headers=headers, data=data)
 
     return process_cs_response(resp.json(), users_by_id)
+
+
+def get_flagged_threads():
+    """
+    Generator function that calls the edX comments service API and yields a
+    thread dictionary for each flagged post.
+    """
+    api_url = settings.CS_URL_BASE + '/api/v1/flagged_threads'
```

Comment:
Was just going for consistency with
https://github.com/Stanford-Online/notifier/pull/4/files#diff-76923973be8920ac2eb626c504268102L141
but this way is probably stylistically better, I'll go ahead and change it to this.